ai_genomics/__init__.py

```diff
@@ -16,6 +16,7 @@ def get_yaml_config(file_path: Path) -> Optional[dict]:
 
 # Define project base directory
 PROJECT_DIR = Path(__file__).resolve().parents[1]
+bucket_name = "ai-genomics"
```

Comment:
This is also [defined as an environment variable](https://github.com/nestauk/ai_genomics/blob/dev/.envrc) you can access with `os` or `dotenv`